scripts/bootstrap.py

```diff
@@ -69,12 +69,12 @@
 except ImportError:
     ez = {}
     if USE_DISTRIBUTE:
-        exec urllib2.urlopen('http://python-distribute.org/distribute_setup.py'
-                         ).read() in ez
+        exec(urllib2.urlopen('http://python-distribute.org/distribute_setup.py'
```

Comment:
It seems to have been updated, yes: https://raw.githubusercontent.com/buildout/buildout/master/bootstrap/bootstrap.py I don't understand the build scripts well enough to test them, so I've reverted my changes to the `scripts` directory in 27ce6398b82d26c6b3a44caa64e7df6199375175.

My primary purpose in this PR was get the test suite to pass on Python 3 and update the trove classifiers so that `pathtools` no longer appears as "not ready" on [the wall of python 3 readiness](http://py3readiness.org/). The build scripts can stay on python 2 until its end of life.